### DISCUSSION TITLE
TuGraph是否支持直接写SST文件，然后将SST文件加载到已存在数据的子图中

### DISCUSSION THREAD

**Discussion by huzhihao1** [TuGraph-family/tugraph-db#617] [Q&A]
前置条件：当前数据库中已存在一张有数据的子图
诉求：通过spark或其他手段，批量写SST文件，是否可以直接将SST文件加载到已有数据的子图中。
我在看TuGraph的源码时，看到在线全量导入时，使用的就是写SST文件，然后将SST等文件信息加载为一张子图；想请教下是否支持将SST等文件信息加载到一张有数据的子图中。

**Comment by qishipengqsp** [comment]
数据导入是使用rocksDB实现排序的逻辑，目前持久化存储TuGraph-DB使用的还是LMDB，所以暂不支持将SST文件load进库。目前我们正考虑支持多存储引擎，可以继续关注一下

**Reply by huzhihao1** [reply]
当前我看了下TuGraph 的在线全量导入接口会调用DoImportOffline，用的就是写SST文件的方式：
![image](https://github.com/user-attachments/assets/5238fb38-a4b4-4ccc-a023-e85b7bdb2cbb)
有几个问题请教下：
在线全量导入是否可以使用SST文件加载到空的子图？
是否可以使用分布式的方式直接写SST文件来做导入加速？
是否有其他可借鉴的并行导入的手段？
